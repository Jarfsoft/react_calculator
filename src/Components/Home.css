.quote {
  margin: 5rem;
  font-size: 1.5rem;
  font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
}
