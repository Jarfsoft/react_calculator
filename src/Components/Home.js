import React from 'react';

import './Home.css';

const Home = () => (
  <div className="quote">
    <h2>Greeting, fellow</h2>
    <p>A react app developed to do some calculations with. Use it!</p>
    <p>This project was made in order to test my abilities as a developer.</p>
    <p>Take a look at the Github repository in the link below.</p>
    <a href="https://github.com/Jarfsoft/react_calculator" target="_blank" rel="noreferrer">Juan Raudales</a>
  </div>
);

export default Home;
