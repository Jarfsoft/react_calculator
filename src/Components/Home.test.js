import React from 'react';
import renderer from 'react-test-renderer';
import Home from './Home';

it('render the home page', () => {
  const home = renderer
    .create(<Home />)
    .toJSON();
  expect(home).toMatchSnapshot();
});
