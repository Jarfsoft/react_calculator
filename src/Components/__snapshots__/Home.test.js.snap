// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render the home page 1`] = `
<div
  className="quote"
>
  <h2>
    Greeting, fellow
  </h2>
  <p>
    A react app developed to do some calculations with. Use it!
  </p>
  <p>
    This project was made in order to test my abilities as a developer.
  </p>
  <p>
    Take a look at the Github repository in the link below.
  </p>
  <a
    href="https://github.com/Jarfsoft/react_calculator"
    rel="noreferrer"
    target="_blank"
  >
    Juan Raudales
  </a>
</div>
`;
